src/api/app/controllers/webui/project_controller.rb

```diff
@@ -532,13 +533,15 @@ def save_meta
       errors << exception.message
     end
 
-    if errors.empty?
-      flash.now[:success] = 'Config successfully saved!'
-      render layout: false, partial: 'layouts/webui/flash', object: flash
-    else
-      flash.now[:error] = errors.compact.join("\n")
-      render layout: false, status: 400, partial: 'layouts/webui/flash', object: flash
-    end
+    status = if errors.empty?
```

Comment:
[nitpick]

I prefer how it was done in [this PR](https://github.com/openSUSE/open-build-service/pull/6373/files#diff-86772a8e6d058f9f138b75ac35927ffdR557)

Reply:
i prefer like that :)

Reply:
I just don't like the flash in the middle, but it's fine. We don't have to go [Mortal Combat style](https://www.youtube.com/watch?v=BY1k1GcZRww) :smile:

Reply:
![](https://66.media.tumblr.com/c178aff4ce616442baa1cd9261385abe/tumblr_mqpqov8t1z1scjl3lo1_500.gif)

Reply:
Rubocop likes more @vpereira option: https://rubocop.readthedocs.io/en/latest/cops_style, we just disabled it because some people in the team didn't like it. So I am with @vpereira

Reply:
Rubocop likes more @vpereira option: https://rubocop.readthedocs.io/en/latest/cops_style, we just disabled it because some people in the team didn't like it. So I am with @vpereira, althoguh to be honest this is not really part of this commit, why is this related to Bootstrap? :confused: